codebase/src/java/disco/org/openlvc/disco/pdu/record/BeamData.java

```diff
@@ -92,6 +92,16 @@ public boolean equals( Object object )
 		return false;
 	}
 	
+	@Override
+	public int hashCode()
```

Comment:
Added a `hashCode()` implementation to keep the linter happy (classes that override `equals` should also override `hashCode`)